dll/cpl/desk/background.c

```diff
@@ -1140,6 +1142,53 @@ SetDesktopBackColor(HWND hwndDlg, PBACKGROUND_DATA pData)
     RegCloseKey(hKey);
 }
 
+static VOID
+OnCustomButton(HWND hwndDlg, PBACKGROUND_DATA pData)
+{
+    TCHAR szCaption[128] = {0};
+    HPROPSHEETPAGE hpsp[1] = {0};
+    PROPSHEETHEADER psh;
+    HPROPSHEETPAGE hPage;
+    PROPSHEETPAGE psp;
+
+    LoadString(hApplet, IDS_DESKTOP_ITEMS, szCaption, sizeof(szCaption) / sizeof(TCHAR));
+
+    ZeroMemory(&psh, sizeof(PROPSHEETHEADER));
+    psh.dwSize = sizeof(PROPSHEETHEADER);
+    psh.dwFlags = PSH_NOAPPLYNOW;
+    psh.hwndParent = GetParent(hwndDlg);
+    psh.hInstance = hApplet;
+    psh.pszCaption = szCaption;
+    psh.phpage = hpsp;
+
+    if (psh.nPages < 1)
```

Comment:
Here, `(psh.nPages < 1)` is always TRUE.

Reply:
Addressed, thanks!